packages/block-editor/src/components/add-block-separator/index.native.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { Text, View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withPreferredColorScheme } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+const AddBlockSeparator = ( { getStylesFromColorScheme } ) => {
```

Comment:
Can we consolidate this new component with https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-list/insertion-point.native.js?

And similarly for the style definitions in https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-list/style.native.scss?

Reply:
I haven't realized that there is such a component. However, it seems like `insertion-point.native.js` is not used at all. I will move the `AddBlockSeparator`  into it to be consistent with web names.